src/app/books/components/books-list/books-list.component.html

```diff
@@ -1,7 +1,8 @@
-<section class="book-list">
+<section class="book-list" *ngIf="booksData">
```

Comment:
I guess your screen will not contain messages in case of empty values. It's not a suitable solution.